af/fastpath/fastpath/normalize.py

```diff
@@ -575,21 +576,24 @@ def iter_yaml_msmt_normalized(data, bucket_tstamp: str, report_fn: str):
     if not header.get("report_id"):
         header["report_id"] = generate_report_id(header)
 
-    for off, entry in blobgen:
+    for off, raw_entry in blobgen:
         esha = headsha.copy()
-        esha.update(entry)
-        esha = esha.digest()
-        entry = yaml.safe_load(entry)
+        esha.update(raw_entry)
+        esha_d = esha.digest()
+        entry = yaml.safe_load(raw_entry)
 
         if not entry:  # e.g. '---\nnull\n...\n'
             continue
         if "test_start_time" in entry and "test_start_time" in header:
             header.pop("test_start_time")
         entry.update(header)
         try:
-            yield normalize_entry(entry, bucket_tstamp, report_fn, esha)
+            d = normalize_entry(entry, bucket_tstamp, report_fn, esha_d)
+            msmt_uid = trivial_id(raw_entry, d)
```

Comment:
Do we care to concatenate the entry header when computing the trivial_id?

Reply:
What's the benefit?

Reply:
I guess that morally a YAML measurement is the header + the entry, but yeah you are probably right that there isn't any real benefit to doing so.

Reply:
We discussed this and it's probably not so helpful to have. Closing.